### PR TITLE
chore(CommunityPermissions): ENS tab simplified according to updated design

### LINKED ISSUE #8818: [Community Permissions] Update ENS tab according to new design
## Description

The ENS tab is going to be simplified (design not ready when defining the issue). Existing implementation must be adjusted to the new design.

https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?node-id=22570%3A493226&t=wgtolxqQkOw6sxsg-0

### PR DESCRIPTION
**Pull request by micieslak**
### What does the PR do

- Fixed ENS icon in "Who holds"
- Fixed locale handling in `AmountInput` (warning was generated, locale was set improperly)
- ENS tab simplified according to updated design

Closes #8818 

### Affected areas
`EnsPanel`, `HoldingsDropdown`, `CommunityNewPermissionsView`

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

Screenshot from 2023-01-18 00-14-47

### CODE REVIEW

File: ui/app/AppLayouts/Chat/controls/community/HoldingsDropdown.qml [hunk 1]
```diff
@@ -232,7 +230,7 @@ StatusDropdown {
                 },
                 State {
                     name: HoldingTypes.Type.Ens
-                    PropertyChanges {target: holdingsTabs; sourceComponent: ensLayout; addOrUpdateButtonEnabled: d.ensReady}
+                    PropertyChanges {target: holdingsTabs; sourceComponent: ensLayout; addOrUpdateButtonEnabled: d.ensReady/*; implicitHeight: 263*/}
```
Comment (caybro) [inline ui/app/AppLayouts/Chat/controls/community/HoldingsDropdown.qml:233]: leftover?

File: ui/imports/shared/controls/AmountInput.qml [hunk 2]
```diff
@@ -34,13 +34,13 @@ Input {
...
 
-    validator:  DoubleValidator {
+    validator: DoubleValidator {
         id: doubleValidator
 
         decimals: root.allowDecimals ? 100 : 0
         bottom: 0
         notation: DoubleValidator.StandardNotation
-        locale: root.locale
+        locale: root.locale.toString()
```
Comment (caybro) [inline ui/imports/shared/controls/AmountInput.qml:43]: Almost :)

Comment (caybro) [reply ui/imports/shared/controls/AmountInput.qml:43]:
```suggestion
        locale: root.locale.name
```

Comment (caybro) [reply ui/imports/shared/controls/AmountInput.qml:43]: https://doc.qt.io/qt-5/qml-qtqml-locale.html#name-prop

**Review by caybro — CHANGES_REQUESTED** [review]
Looks good in general, 2 minor things though

File: ui/imports/shared/controls/AmountInput.qml (hunk 2 above)
Comment (micieslak) [reply ui/imports/shared/controls/AmountInput.qml]: haha :man_facepalming:, fixed :)

**Review by caybro — APPROVED** [review]

**Review by noeliaSD — APPROVED** [review]
LGTM! Tested!

**Review by glitchminer — COMMENTED** [review]
Issues found

Input is flagged as invalid with every second character.

https://user-images.githubusercontent.com/50769329/213485266-2ccd09d9-6691-4d6c-802f-e02ae438c623.mp4

**Review by glitchminer — APPROVED** [review]
Tested, approved with caveat:

Requirement for input validation and regex to be further refined and tested. Currently it's possible to add items that are not on the .eth domain. Character limits would also need reviewed.
image

Other entries appear OK: 
image

**Pull request merged**